kernel/core/io/file.cpp

```diff
@@ -3899,6 +3899,33 @@ namespace NumeRe
         // Empty destructor
     }
 
+    /////////////////////////////////////////////////
+    /// \brief Check if the given format string corresponds
+    /// to a date-time format. This function looks for
+    /// the presence of characters 'm', 'd', 's', 'h' (and their
+    /// uppercase equivalents) in the format string. These
+    /// characters are typically used in date-time formatting
+    /// in an Excel documents xl/style.xml file.
+    ///
+    /// \param formatString The string representing the format
+    /// code to be checked.
+    ///
+    /// \return true if the formatString is a date-time format,
+    /// false otherwise.
+    ///
+    /////////////////////////////////////////////////
+    bool isDateTimeFormat(const std::string& formatString)
+    {
+        // List of characters to search for
+        const char charsToFind[] = {'m', 'd', 's', 'h', 'M', 'D', 'S', 'H'};

```

Comment:
For some reasons I cant convert to Draft mode. I need to add j and J here. Maybe adding English variants like y or Y also makes sense for this case.

Is this function okay as is? or best written in .hpp?